.gitignore

```diff
@@ -3,3 +3,5 @@ node_modules/
 
 # Optional eslint cache
 .eslintcache
+
+.idea
```

Comment:
This should be in global gitignore on your system. See https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Here's a good example for JetBrains IDE: https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore